packages/patternfly-4/react-charts/src/components/ChartBar/examples/ChartBar.md

```diff
@@ -10,14 +10,15 @@ import './chart-bar.scss';
 
 Note: PatternFly React charts live in its own package at [@patternfly/react-charts](https://www.npmjs.com/package/@patternfly/react-charts)!
 
+## Introduction
```

Comment:
This paragraph is used in most chart examples. It appears they should all have an `## Introduction` tag?